src/main/java/com/minecolonies/blockout/views/SwitchView.java

```diff
@@ -43,17 +43,24 @@ public void parseChildren(@NotNull final PaneParams params)
         }
     }
 
-    public void setView(final String name)
+    /**
+     * Switch current view to view with id given as param
+     * 
+     * @param name id of view
+     * @return true if view of given name was found, else false
+     */
+    public boolean setView(final String name)
```

Comment:
Where is the return value used?

Reply:
In the upcoming guidebook, kinda sorry I just grabbed all changes in the blockout directory...

Reply:
Maybe is can use findPaneOfTypeByID(id, SwitchView.class).getChildren().contains() but I think this more suitable for some bigger switchViews, maybe can also throw a warning into log

Reply:
Okey